.circleci/config.yml

```diff
@@ -168,6 +168,50 @@ jobs:
       - runtests:
           platform: conscrypt
 
+  okcurl-graal:
+    docker:
+      - image: circleci/openjdk:11.0.3-jdk-stretch
+
+    environment:
+      JVM_OPTS: -Xmx1g
+      TERM: dumb
+
+    steps:
+      - checkout
+
+      - restore_cache:
+          keys:
+            # restore compilation and wrapper from master build
+            - v4-master-compile
+
+      - run:
+          name: Download Graal
+          command: ./gradlew :okcurl:extractGraalTooling
+          environment:
+            GRADLE_OPTS: -Dorg.gradle.daemon=false -Dorg.gradle.workers.max=3 -Xmx1G
+
+      - run:
+          name: Debug
+          command: find ~/.gradle/caches/com.palantir.graal
+
+      - run:
+          name: Install native-image
+          command: ~/.gradle/caches/com.palantir.graal/19.0.2/graalvm-ce-19.0.2/Contents/Home/bin/gu install native-image
+
+      - run:
+          name: Build nativeImage
+          command: ./gradlew :okcurl:nativeImage
```

Comment:
Fails in CircleCI, but working locally.  Might ignore circleci support for now :(

```
> Task :okcurl:nativeImage
Build on Server(pid: 490, port: 44279)*
[okcurl:490]    classlist:   2,749.25 ms
[okcurl:490]        (cap):     545.02 ms
[okcurl:490]        setup:     860.60 ms
Error: Basic header file missing (<zlib.h>). Make sure headers are available on your system.
Error: Use -H:+ReportExceptionStackTraces to print stacktrace of underlying exception
Error: Image build request failed with exit status 1
```